h2o-py/tests/testdir_munging/unop/pyunit_cor.py

```diff
@@ -9,22 +8,18 @@
 # If any categorical columns, throw an error
 ##
 
-
-
-
 import numpy as np
+import pandas as pd
 
 def cor_test():
-
-
-
     iris_h2o = h2o.import_file(path=pyunit_utils.locate("smalldata/iris/iris.csv"))
     iris_np = np.genfromtxt(pyunit_utils.locate("smalldata/iris/iris.csv"),
                             delimiter=',',
-                            skip_header=1,
+                            skip_header=-1,
```

Comment:
This was a bug. It skipped the first line. Therefore the `assert max < .006, "expected equal correlations"`. Now, the difference is 0 !